system/CLI/Commands.php

```diff
@@ -77,39 +74,38 @@ public function run(string $command, array $params)
 	 */
 	public function getCommands()
 	{
-		$this->discoverCommands();
-
 		return $this->commands;
 	}
 
 	/**
 	 * Discovers all commands in the framework and within user code,
 	 * and collects instances of them to work with.
+	 *
+	 * @return void
 	 */
 	public function discoverCommands()
 	{
-		if (! empty($this->commands))
+		if ($this->commands !== [])
 		{
 			return;
 		}
 
-		$files = service('locator')->listFiles('Commands/');
+		$locator = Services::locator();
```

Comment:
Lonnie wants us to start using the helper function. 
```suggestion
		$locator = service('locator');
```